ui/admin/ember-cli-build.js

```diff
@@ -34,8 +34,9 @@ module.exports = function (defaults) {
       },
     },
     // TODO: Update to 4.12 when deprecations are resolved
+    //   as multiple things break when forcing compatability on > 4.6
     emberData: {
-      compatWith: '4.4',
+      compatWith: '4.6',
```

Comment:
It'd be good to get this to 4.12 at some point since there are supposedly [performance increases](https://blog.emberjs.com/ember-released-4-7/) with ember data in 4.7 and beyond